labs/behaviors/focusable.ts

```diff
@@ -77,10 +77,12 @@ export function mixinFocusable<T extends MixinBase<LitElement>>(
     constructor(...args: any[]) {
       super(...args);
       this[isFocusable] = true;
-      queueMicrotask(() => {
-        this[hasConstructed] = true;
-        this[updateTabIndex]();
-      });
+    }
+
+    connectedCallback() {
```

Comment:
Full disclosure: this may be a harder change to land. Changing timings from a microtask to a task could be hard enough with test timings to consider. Moving to another lifecycle entirely might result in more breakages.

Won't know until we do some more internal testing though